src/main/java/com/icthh/xm/tmf/ms/resource/security/SpringSecurityAuditorAware.java

```diff
@@ -1,10 +1,11 @@
 package com.icthh.xm.tmf.ms.resource.security;
 
 import com.icthh.xm.tmf.ms.resource.config.Constants;
-import java.util.Optional;
 import org.springframework.data.domain.AuditorAware;
 import org.springframework.stereotype.Component;
 
+import java.util.Optional;
+
```

Comment:
Unnecessary changes, according checkstyle imports orders must be the same as in old version